var/spack/repos/builtin/packages/spfft/package.py

```diff
@@ -38,7 +41,10 @@ def cmake_args(self):
         if self.spec.satisfies('+gpu_direct'):
             args += ["-DSPFFT_GPU_DIRECT=On"]
         if self.spec.satisfies('+cuda'):
-            args += ["-DSPFFT_BACKEND=CUDA"]
-        if self.spec.satisfies('+cuda'):
-            args += ["-DSPFFT_BACKEND=CUDA"]
+            args += ["-DSPFFT_GPU_BACKEND=CUDA"]
```

Comment:
Was this wrong before or is the option dependent on the version of the package used (`-DSPFFT_BACKEND` vs. `-DSPFFT_GPU_BACKEND`?

Reply:
Yes, this was wrong before.

Reply:
Thanks for the confirmation.